### PR TITLE
Add support for Bun

### LINKED ISSUE #25: [feature request] Add support for the bun package manager
Executing `bunx update-browserslist-db@latest` results in the following error:

```bash
update-browserslist-db: No lockfile found. Run "npm install", "yarn install" or "pnpm install"
```

Bun uses a binary lockfile called `bun.lockb` and at the time of writing has no built-in method yet to update individual packages.

### PR DESCRIPTION
**Pull request by kytta**
This PR adds support for Bun:

- info is still queried from NPM, as Bun does not have an `info` command
- update is done via `bun update`
- tests use `bun pm ls --all` to query the lockfile

Closes #25

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -264,4 +264,18 @@ test('updates caniuse-lite for pnpm', async () => {
...
+  await chdir('update-bun', 'package.json', 'bun.lockb')
+  match(
+    runUpdate(),
+    `Latest version:     ${caniuse.version}\n` +
+      'Updating caniuse-lite version\n' +
+      '$ bun update caniuse-lite\n' +
+      'caniuse-lite has been successfully updated\n'
+  )
+
+  let pmls = execSync('bun pm ls --all', { env: { ...process.env, NO_COLOR: "1", FORCE_COLOR: "0" } }).toString()
```
Comment (kytta) [inline test/index.test.js:277]:
```suggestion
  let pmls = execSync('bun pm ls --all', { env: { ...process.env, FORCE_COLOR: "0", NO_COLOR: "1" } }).toString()
```

**Comment by ai** [comment]
I will combine both your PRs and both of you will be mentioned in Cult of Martians.

**Comment by ai** [comment]
Do you have Twitter account to mention you in release tweet?

**Comment by ai** [comment]
Do you have Twitter account to mention you in release tweet?

**Comment by kytta** [comment]
> Do you have Twitter account to mention you in release tweet?

Yes, it's https://twitter.com/kyttawashere (albeit inactive)

**Pull request merged**